packages/plugins/other/jsdoc-operations/src/index.ts

```diff
@@ -0,0 +1,118 @@
+import { PluginFunction } from '@graphql-codegen/plugin-helpers';
+import { printSchema, parse, visit } from 'graphql';
+
+type NamedType = {
+  kind: 'NamedType';
+  value: string;
+  isRequired?: boolean;
+};
+
+type ListType = {
+  kind: 'ListType';
+  item: NamedType;
+  isRequired?: boolean;
+};
+
+type Property = {
+  name: string;
+  type: ListType | NamedType;
+};
+
+type TypeDef = {
+  name: string;
+  operation: string;
+  properties?: Array<Property>;
+  types?: Array<NamedType>;
+};
+
+const transformScalar = (scalar: string) => {
+  const scalarMap = {
```

Comment:
You can use `@graphql-codegen/visitor-plugin-common` and import it (`DEFAULT_SCALARS`) from there